apps/remote_control/lib/lexical/remote_control/build/state.ex

```diff
@@ -257,7 +263,8 @@ defmodule Lexical.RemoteControl.Build.State do
     end
 
     with_progress "mix deps.compile", fn ->
-      Mix.Task.run("deps.safe_compile", ~w(--skip-umbrella-children))
+      deps_compile = if System.version() >= "1.15", do: "deps.compile", else: "deps.safe_compile"
```

Comment:
I _really_ dislike this syntax; use the full if syntax. It's so much more readable.
Also, you cannot compare strings and versions like this, there's an entire module for versions
```suggestion
   compile_task = 
      if Version.match?(System.version(), ">= 1.15") do 
        "deps.compile"
      else
        "deps.safe_compile"
      end
```

Reply:
we should not use `Version.match?(System.version(), ">= 1.15") `, we should use `~> 1.15`

```elixir
iex(1)> Version.match?(System.version(), ">= 1.15")
** (Version.InvalidRequirementError) invalid requirement: ">= 1.15"
    (elixir 1.15.3) lib/version.ex:441: Version.parse_requirement!/1
    (elixir 1.15.3) lib/version.ex:287: Version.match?/3
    iex:1: (file)
iex(1)> Version.match?(System.version(), "~> 1.15")
true
iex(2)> Version.match?("1.16.0", "~> 1.15")
true
```